Source/SplineFollow/Public/Components/SFSplineMarkers.h

```diff
@@ -215,6 +225,13 @@ struct SPLINEFOLLOW_API FSFSplineMarker
         Name = TEXT( "Marker" );
     }
 
+    explicit FSFSplineMarker( const FName & Name, uint8 ItIsEnabled, const FSFSplineMarkerInfos & Infos, const TObjectPtr< USFSplineMarkerObject > & Object ) :
```

Comment:
no need for `explicit` when the ctor has more that 1 argument

Reply:
coding standard (snake_case)

Reply:
done